src/Types.jl

```diff
@@ -178,7 +178,17 @@ function projectfile_path(env_path::String; strict=false)
 end
 
 function manifestfile_path(env_path::String; strict=false)
-    for name in Base.manifest_names
+    man_names = Base.manifest_names isa Tuple ? Base.manifest_names : Base.manifest_names()
```

Comment:
```suggestion
    @static if Base.VERSION >= v"1.9.0-"
        man_names = Base.manifest_names()
    else
        man_names = Base.manifest_names
    end
```

Reply:
This will have to follow. We can't merge this kind of catch before it's merged in Base

Reply:
Just as a short-term stop-gap, how about this?

```suggestion
    @static if Base.manifest_names isa Tuple
        man_names = Base.manifest_names()
    else
        man_names = Base.manifest_names
    end
```

Then we can replace it with the `Base.VERSION` check once the Base PR is merged and the nightly binaries catch up.

Reply:
Or, if you want to keep the ternary:

```suggestion
    man_names = @static Base.manifest_names isa Tuple ? Base.manifest_names : Base.manifest_names()
```